src/app/components/SalonOwner/SalonOwner.tsx

```diff
@@ -2,35 +2,50 @@ import React from "react";
 import Image, { StaticImageData } from "next/image";
 import owner from "@images/owner.png";
 import Container from "../Share/Container";
+import Button from "../Share/Button";
 
 const SalonOwner: React.FC = () => {
   return (
     <section id="owner" className="pb-[48px] pt-[70px] desktop:pt-[88px]">
       <Container>
-        <h2 className="mb-4 text-left font-bold">O właścicielce salonu</h2>
-        <div className="mb-4 flex justify-center">
-          <Image
-            src={owner as StaticImageData}
-            alt="Katarzyna Bober"
-            className="h-fit w-fit object-cover"
-          />
-        </div>
-        <div className="text-center">
-          <h3 className="text-md text-left font-semibold">Katarzyna Bober</h3>
-          <p className="mb-4 text-left text-sm text-gray-700">
-            Technik weterynarii
-          </p>
-          <p className="mb-4 text-left text-sm text-gray-700">
-            Katarzyna Bober, technik weterynarii i właścicielka salonu
-            groomerskiego Spoon Dog od 2011 roku, zdobyła swoje doświadczenie w
-            Opolskim Schronisku dla Bezdomnych Zwierząt oraz studiując
-            zootechnikę we Wrocławiu. Posiada dyplom groomera ze szkoły Pani Ewy
-            Grabowskiej i certyfikat Emmy Pet, co pozwala jej dbać o higienę i
-            zdrowie psów. Jako ambasador Blue Dog edukuje dzieci i dorosłych.
-          </p>
-          <button className="bg-yellow-400 rounded-full px-4 py-2 font-semibold text-black">
-            Dowiedz się więcej
-          </button>
+        <p className="mb-3 border-b border-x-greyLightLines pb-1 text-14 leading-6 text-greyFont desktop:border-b-0 desktop:pb-0 desktop:text-16">
+          O nas / Katarzyna Bober
+        </p>
+        <h2 className="mb-10 text-left text-26 font-bold leading-5 tracking-[-.52px] desktop:text-32 desktop:font-medium desktop:tracking-[-.64px]">
+          O właścicielce salonu
+        </h2>
+        <div className="desktop:flex desktop:flex-row-reverse desktop:justify-between">
+          <div className="mb-10 flex h-[248px] w-[342px] justify-center desktop:h-[391px] desktop:w-[540px]">
+            <Image
+              src={owner as StaticImageData}
+              alt="Katarzyna Bober"
+              className="h-full w-full object-cover"
+              sizes="(max-width: 1439px) 342px, (min-width: 1440px) 540px"
+            />
+          </div>
+          <div className="text-center desktop:h-[430px] desktop:w-[486px]">
+            <h3 className="text-md mb-1 text-left text-20 font-medium leading-6 tracking-[-.4px] desktop:mb-4 desktop:text-26 desktop:font-normal desktop:-tracking-normal">
+              Katarzyna Bober
+            </h3>
+            <p className="mb-4 text-left text-16 text-sm leading-6 tracking-[-.32px] desktop:mb-4 desktop:text-18 desktop:font-medium desktop:tracking-normal">
```

Comment:
Delete the redundant class, text-sm, when text-16 is already in use, which automatically sets the correct size :)